package.json

```diff
@@ -10,7 +10,8 @@
     "ios": "react-native run-ios && react-native log-ios",
     "android": "react-native run-android && react-native log-android",
     "android:install": "react-native run-android && adb install android/app/build/outputs/apk/app-debug.apk && react-native log-android",
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "test": "tap test/*.js"
```

Comment:
Yes please! I was experimenting with `tape` vs `tap` and it looks like I didn't make my usage consistent across my PR